src/services/qrService.ts

```diff
@@ -387,6 +394,26 @@ const connectOpenId4VcIssuance = async (args: IQrDataArgs): Promise<void> => {
                 },
               }
             );
+
+            const navigationState = navigationRef.current.getRootState();
+            const { routes, index } = navigationState;
+
+            console.log('   ')
```

Comment:
Do we still need these console.logs?

Reply:
nah, this was some test code to see if i could get the active screens within the stack.

code has been removed